keepassxc-browser/options/options.html

```diff
@@ -279,6 +280,52 @@ <h3 id="myModalLabel">Remove specified credential fields?</h3>
        </div>
       </div>
 
+      <!-- Ignored sites -->
+      <div class="tab" id="tab-ignored-sites">
+        <h2>Ignored sites</h2>
+        <hr />
+        <p>
+          Sites in this list are ignored when new credentials are detected.
+          <br />
+          Just go to the page with new credentials and click on the blinking KeePassXC-Browser-Icon or the notification, now select <em>Never ask for this page</em>.
```

Comment:
`Go to the page with new credentials, click the blinking KeePassXC-Browser icon or the notification and select <em>Never ask for this page</em>.`